apps/cookbook/src/app/showcase/link-showcase/link-showcase.component.html

```diff
@@ -0,0 +1,43 @@
+<p>
+  Links are used for navigation between pages. They can be standalone links used on their own or
+  inline links placed within a sentence.
+</p>
+<p>
+  A link should be used for simple navigation. If a user triggers an action, makes a choice, or in
+  any way manipulate data, use a <a routerLink="/home/showcase/button">button</a> instead.
+</p>
+<p>
+  Rather than providing a component for links, Kirby simply provides custom styles for the
+  <code>&lt;a&gt;</code> tag, that gives a consistent look and feel of links. Links have an
+  underline, and inherit the text color of the surrounding elements. The color of visited links does
+  not change.
+</p>
+
+<h2>Sizes</h2>
+<p>
+  Any font-class from <a routerLink="/home/showcase/fonts">Kirby Fonts</a> can be used with links.
+  For accessability reasons, it is not recommended to use links when the clickable text is smaller
+  than 12px (extra small).
+</p>
+
```

Comment:
I'd expect that links inherit their font-size like they normally do and I believe that _is_ how they are styled in Kirby.

Consider to make that clear by adding a sentence before "Any font-class from [...]" . For "stand-alone" links it's probably good to know that you can use "Kirby sizes" directly on the link element.